mocca-client/src/main/java/com/paypal/mocca/client/MoccaFeignEncoder.java

```diff
@@ -29,6 +34,16 @@
 class MoccaFeignEncoder implements Encoder {
 
     private final MoccaSerializer moccaSerializer = new MoccaSerializer();
+    Validator validator;
+
+    MoccaFeignEncoder() {
+        try {
+            Validator validator = Validation.buildDefaultValidatorFactory().getValidator();
```

Comment:
Will the `MoccaFeignEncoder` always instantiate its own validator?

Reply:
That's the idea. If the user adds a validation implementation like hibernate to the classpath the validator will be non-null, otherwise it will be null and ignored.